packages/dashboards/tests/acceptance/print-dashboard-test.js

```diff
@@ -1,7 +1,11 @@
 import { test } from 'qunit';
 import moduleForAcceptance from '../../tests/helpers/module-for-acceptance';
 
-moduleForAcceptance('Acceptance | print dashboard');
+moduleForAcceptance('Acceptance | print dashboard', {
+  afterEach() {
+    server.shutdown();
```

Comment:
Where is these server instances instantiated?

Reply:
I'm not sure exactly, but every acceptance test was printing a warning message about having multiple Pretender instances running.

Reply:
I think it's this server instance
https://github.com/samselikoff/ember-cli-mirage/blob/2703e767bcb3e47c8694691aae2c076c45fe0d6c/app/initializers/ember-cli-mirage.js#L16

Reply:
We probably won't need these server.shutDown() calls once mirage is upgraded
https://github.com/samselikoff/ember-cli-mirage/issues/915